cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -107,11 +137,19 @@ func createPoolBuilder(cStorPool *apis.CStorPool) []string {
 
 	poolNameUID := string(PoolPrefix) + string(cStorPool.ObjectMeta.UID)
 	createAttr = append(createAttr, poolNameUID)
+	diskList := cStorPool.Spec.Disks.DiskList
+	poolType := cStorPool.Spec.PoolSpec.PoolType
 	// To generate mirror disk0 disk1 mirror disk2 disk3 format.
-	for i, disk := range cStorPool.Spec.Disks.DiskList {
-		if cStorPool.Spec.PoolSpec.PoolType == "mirrored" && i%2 == 0 {
+	for i, disk := range diskList {
+		if poolType == mirrored && i%2 == 0 {
 			createAttr = append(createAttr, "mirror")
 		}
+		if i == 0 && poolType == raidz {
```

Comment:
Similar to mirror being group of 2, here, raidz are groups of 3, and raidz2 are group of 6. So, raidz and raidz2 keywords need to be repeated.

Reply:
ok